src/chrome/chromeDebugSession.ts

```diff
@@ -202,6 +220,18 @@ export class ChromeDebugSession extends LoggingDebugSession {
     private sendUnknownCommandResponse(response: DebugProtocol.Response, command: string): void {
         this.sendErrorResponse(response, 1014, `[${this._extensionName}] Unrecognized request: ${command}`, null, ErrorDestination.Telemetry);
     }
+
+    private fillErrorDetails(properties: ITelemetryPropertiesPayload, e: any) {
```

Comment:
Should this set exceptionType?

Reply:
Maybe not, caller can customize and set that